edx_exams/apps/api/serializers.py

```diff
@@ -1,4 +1,33 @@
-# Serializers that can be shared across multiple versions of the API
-# should be created here. As the API evolves, serializers may become more
-# specific to a particular version of the API. In this case, the serializers
-# in question should be moved to versioned sub-package.
+"""
+Serializers for the edx-exams API
+"""
+from rest_framework import serializers
+
+from edx_exams.apps.core.models import Exam
+
+
+class ExamSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the Exam Model
+    """
+
+    exam_name = serializers.CharField(required=True)
+    course_id = serializers.CharField(required=True)
+    content_id = serializers.CharField(required=True)
+    time_limit_mins = serializers.IntegerField(required=True)
+    due_date = serializers.DateTimeField(required=False, format=None)
+    exam_type = serializers.CharField(required=True)
```

Comment:
Do we need extra validation on exam_type? Right now it could be any string. I feel like it should be limited to a set, but I'm not sure what that set should contain.

Reply:
That can be done later. Maybe create a jira ticket and evaluate the need for that validation later?